### PR TITLE
[docs] translate mysql cdc doc to Chinese.

### PR DESCRIPTION
**Pull request by rookiegao**
Since the debezium version has not been upgraded to 1.6 when it was released in 2.2, the link about debezium in this document is still 1.5. In addition, the version of CDC connector is unified as 2.2.1

### CODE REVIEW

File: docs/content/connectors/mysql-cdc(ZH).md [hunk 1]
```diff
@@ -0,0 +1,869 @@
...
+为了设置 MySQL CDC 连接器，下表提供了使用构建自动化工具（如 Maven 或 SBT ）和带有 SQL JAR 包的 SQL 客户端的两个项目的依赖关系信息。
+
+### Maven dependency
+
+```
+<dependency>
+  <groupId>com.ververica</groupId>
+  <artifactId>flink-connector-mysql-cdc</artifactId>
+  <!-- 请使用已发布的版本依赖，snapshot版本的依赖需要本地自行编译。 -->
+  <version>2.3-SNAPSHOT</version>
```
Comment (leonardBang) [inline docs/content/connectors/mysql-cdc(ZH).md:23]:
```suggestion
  <version>2.2.1</version>
```

File: docs/content/connectors/mysql-cdc(ZH).md [hunk 2]
```diff
@@ -0,0 +1,869 @@
...
+```
+**注意:** 在 `scan.incremental.snapshot.enabled` 参数已启用时（默认情况下已启用）时，不再需要授予 reload 权限。
+
+3. 刷新用户权限：
+
+```sql
+mysql> FLUSH PRIVILEGES;
+```
+
+查看更多用户权限问题请参考 [权限说明](https://debezium.io/documentation/reference/1.5/connectors/mysql.html#mysql-creating-user).
```
Comment (leonardBang) [inline docs/content/connectors/mysql-cdc(ZH).md:53]:
```suggestion
查看更多用户权限问题请参考 [权限说明](https://debezium.io/documentation/reference/1.6/connectors/mysql.html#mysql-creating-user).
```

File: docs/content/connectors/mysql-cdc(ZH).md [hunk 3]
```diff
@@ -0,0 +1,869 @@
...
+           请查阅 <a href="#启动模式">启动模式</a> 章节了解更多详细信息。</td>
+    </tr> 
+    <tr>
+      <td>server-time-zone</td>
+      <td>optional</td>
+      <td style="word-wrap: break-word;">UTC</td>
+      <td>String</td>
+      <td>数据库服务器中的会话时区， 例如： "Asia/Shanghai". 
+          它控制 MYSQL 中的时间戳类型如何转换为字符串。
+          更多请参考 <a href="https://debezium.io/documentation/reference/1.5/connectors/mysql.html#mysql-temporal-types"> 这里</a>.</td>
```
Comment (leonardBang) [inline docs/content/connectors/mysql-cdc(ZH).md:221]:
```suggestion
          更多请参考 <a href="https://debezium.io/documentation/reference/1.6/connectors/mysql.html#mysql-temporal-types"> 这里</a>.</td>
```

**Review by leonardBang — COMMENTED** [review]
Thanks @rookiegao for the contribution, I left several minor comments

File: docs/content/connectors/mysql-cdc(ZH).md (hunk 3 above)
Comment (rookiegao) [reply docs/content/connectors/mysql-cdc(ZH).md:221]: As far as I know, debezium in release 2.2 is still 1.5. Should we still link to debezium 1.6 in the 2.2 document？
@leonardBang

Comment (leonardBang) [reply docs/content/connectors/mysql-cdc(ZH).md:221]: @rookiegao Good catch，you're right that we should use 1.5 link

**Review by leonardBang — APPROVED** [review]

**Pull request merged**